de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/nodes/ScriptHandler.java

```diff
@@ -156,7 +157,13 @@ public static ScriptHandler createHandler(String script_type, List<String> packa
       if (type.startsWith("r")) {
         handler = new RScriptHandler(packages);
       } else if (type.startsWith("py")) {
-        handler = new PythonScriptHandler();
+        if(type.startsWith("python 2")) {
+          handler = new PythonScriptHandler(PythonVersion.PYTHON2);
+        }else if(type.startsWith("python 3")){

```

Comment:
We need a whitespace before any open curly brace. https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace